code/datums/mutations/sight.dm

```diff
@@ -95,6 +96,12 @@
 
 	user_mob.adjustOrganLoss(ORGAN_SLOT_EYES, eye_damage)
 
+/datum/mutation/human/thermal/on_losing(mob/living/carbon/human/owner, mob/user = usr)
```

Comment:
why the user arg?

Reply:
https://github.com/tgstation/tgstation/blob/55b92786dcf0f9c5172efd111a6ecbf83ec0dae2/code/datums/mutations/_mutations.dm#L97

Reply:
ahhh because that was used for the to_chat thing i had, its not nesscary now that its moved to the on loss var